### DISCUSSION TITLE
A "Heat" mechanic

### DISCUSSION THREAD

**Discussion by hyperjuni** [ErinaSugino/Starbound-SBR-Mod#29] [Ideas]
The idea of being "in Heat" was brought up on my (Neki) Discord server, and I figured it could be a fun and quite canon mechanic for a lot of species. My idea is something like: for every set period of gameplay your character receives a debuff, which takes intercourse to get rid of.

Example:
Every hour the character receives a debuff;
The debuff would last for around 20-30 minutes;
Movement/jump speeds get randomly reduced momentarily;
The character is making "funny noises" when/while that happens;
The only way to remove the debuff is to perform intercourse with a compatible mate :3

**Comment by ErinaSugino** [comment]
Making that a player mechanic is easy, and would tie into #9 
As for making it work on NPCs that would be more complicated. Having random bandits in the wild (when they exist for a heat-based race) randomly be affected by debuffs seems bad. Would also throw up the question if, and if so how, it ties into the normal arousal mechanic. Do NPCs in heat just immediately seek out sex (i.e. force set arousal of 100)? How far does "a compatible mate" track?
